javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,39 @@
+var width = 10;
+var height = 5;
+var area = width * height;
+console.log('value of area: ' + area);
+console.log('type of area: ' + typeof area);
+
+var bill = 3;
+var payment = 10;
+var change = payment - bill;
+console.log('value of change: ' + change);
+console.log('type of change: ' + typeof change);
+
+var quizzes = 6;
+var midterm = 9;
+var final = 10;
+var grade = (quizzes + midterm + final) / 3;
+console.log('value of grade: ' + grade);
+console.log('type of grade: ' + typeof grade);
+
+var firstName = 'Evan';
+var lastName = 'Hong';
+var fullName = 'Evan ' + 'Hong'
+console.log('value of fullName: ' +  fullName);
+console.log('type of fullName: ' + typeof fullName);
+
+var pH = 9
+var isAcidic = pH < 7
+console.log('value of isAcidic: ' +  isAcidic);
+console.log('type of is Acidic: ' + typeof isAcidic);
+
+var headCount = 8
+var isSparta = headCount==300
```

Comment:
You should be using strictly equal here. So you're missing a `=`. It should be `===`